script.js

```diff
@@ -1,28 +1,32 @@
 $(document).ready(function(){
 
-
-
-
   var lightSeq = [];
   var turn = 0;
-  var attempts = 0;
-
+  var attempt = 1;
+  var strictMode = false;
```

Comment:
I recommend using `const` instead of `var` when possible - it has a more consistent meaning in the language and prevents possible scoping bugs.